mappings/net/minecraft/world/gen/feature/ReplaceBlobsFeature.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_5155 net/minecraft/world/gen/feature/ReplaceBlobsFeature
```

Comment:
Why this rename? The registry name was not changed (same for the config class)

Reply:
It's called "netherrack replace blobs". Minecraft uses it to replace netherrack, but you can also use a different target. So I removed that from the name to make it shorter and less confusing.

Reply:
If this `netherrack_replace_blobs` is returned by a method in this feature class, then this class should be called `NetherrackReplaceBlobsFeature` (look at advancement criterions). If this is registered like `Registry.register("netherrack_replace_blobs", new ReplaceBlobsFeature())` then `ReplaceBlobsFeature` would be better.

Reply:
It's registered like that in `Feature`.